src/styles/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
```

Comment:
The file uses CSS instead of SCSS. Please convert this file to SCSS file (stars.scss) and update the styles using SCSS syntax.